src/support.js

```diff
@@ -431,7 +425,11 @@ function loadWebAssemblyModule(binary, flags) {
     table.grow(tableSize);
     assert(table === originalTable);
     // zero-initialize memory and table
-    // TODO: in some cases we can tell it is already zero initialized
```

Comment:
Maybe the TODO could be a TODO to use `calloc()` which some day might not zero out if it knows the memory has yet to be used? I think that's still possible, but we never seem to do it so I guess it's not that important...